llvm/lib/Target/EraVM/EraVMInstrFormats.td

```diff
@@ -346,21 +428,59 @@ class IBinaryS<EraVMOpcode opcode,
   let Imm1 = dst0{19-4};
 }
 
-class Irr_r<EraVMOpcode opcode,
-            mod_swap swap_operands, mod_set_flags silent,
-            dag outs, dag ins,
-            string asmstr,
-            list<dag> pattern>
-  : IBinaryR<opcode, SrcReg, swap_operands, silent, outs, ins, asmstr, pattern> {
+// Mix-in classes to hook standard fields for different combinations
+// of input operands.
+
+class SrcRRFields {
```

Comment:
I do not like `Fields` as naming. Maybe just `Operands`? or something like this? After all, the mixin is to indicate the particular instruction has said operands.

Reply:
Renamed `SrcXXFields` to `SrcOperandsXX` according to the intention instead of the implementation details, thanks.